index.js

```diff
@@ -155,7 +162,17 @@ module.exports = function(sails) {
       before: {
         '/*': function(req, res, next) {
           if (injectRequestLogger) {
-            req.log = _this.logger.child({req: req}, true);
+
+            var options = {req: req};
+
+            if (generateRequestId) {
+
+              // jscs: disable
+              options.req_id = uuid.v4(); // jshint ignore:line
```

Comment:
What do you think about including this in the `req` itself? `req.id = options.req_id = uuid.v4()`. That way the request id is accessible from other contexts, like being able to pass it along with other requests throughout the system.